public/api-docs/v1/change_requests.yaml

```diff
@@ -0,0 +1,144 @@
+"/api/v1/planning_applications/{planning_application_id}/change_requests":
+  get:
+    summary: Retrieves all determined planning applications
+    security:
+      - bearerAuth: []
+    parameters:
+      - in: path
+        name: planning_application_id
+        required: true
+        schema:
+          type: integer
+          minimum: 1
+        description: The planning application ID
+      - in: query
+        name: change_access_id
+        schema:
+          type: string
+        required: true
+        description: A unique access code for this planning application
+
+    responses:
+      "200":
+        description: All change requests
+        content:
+          application/json:
+            examples:
+              Full:
+                value:
+                  data:
+                    - id: 1
+                      type: description_change_request
+                      proposed_description: A proposed new description
+                      state: open
+                      response_due: "2020-05-14T05:18:17.540Z"
+                      approved: true
+                      rejection_reason: nil
+
+? "/api/v1/planning_applications/{planning_application_id}/change_requests/{change_request_id}"
```

Comment:
This should be at
`/api/v1/planning_applications/{planning_application_id}/description_change_requests`
When we add different types of change request, then we need to be able to know which table we're looking up.

Reply:
`change_request_id` doesn't need to be added to the path either, as it is already specified in the query params below

Reply:
> Much improved! Few more things to update, and I didn't get a response about either removing the change request show API action or adding some tests for it.

I just removed it? I forgot to take it out of the routes but I took it out of the controller and deleted all tests as well as the show jsonb file